app/components/ui/BookmarkModal/BookmarkModal.module.scss

```diff
@@ -0,0 +1,190 @@
+@import "~styles/utils/_helpers.scss";
+
+.bookmarkModal {
+  width: 600px;
+  padding: 33px;
+  @media screen and (max-width: $break-tablet-s) {
+    max-width: 600px;
+    width: 100vw;
```

Comment:
Should this just be `max-width: 100vw` or `width: 100vw`? I'm not sure if setting both to different values makes sense.

Reply:
Ah, I see, I didn't realize that the breakpoint was larger than 600px. In that case, would it make sense to move both the `max-width: 600px` and `width: 100vw` out of the media query, and into the original class? The original class also currently defines `width: 600px`, but maybe it'd be more uniform to have all these styles apply unconditionally, but where the `width: 100vw` kind of replaces the need for the media query because `vw` is a viewport-width-specific unit.

No need to spend too much more time on this, though.